packages/core/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { makeStyles } from '@material-ui/core';
+import { BackstageTheme } from '@backstage/theme';
+import React, { FC, useCallback } from 'react';
+import { useTable, useGlobalFilter, useSortBy } from 'react-table';
+
+import { FixedSizeList } from 'react-window';
+import MUITable from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+
+import TextField from '@material-ui/core/TextField';
+
+//const useStyles = makeStyles<typeof BackstageTheme>(theme => ({ }));
+type GlobalFilterProps = {
+  setGlobalFilter: any;
+  preGlobalFilteredRows: any;
+  globalFilter: any;
+};
+
+const GlobalFilter: FC<GlobalFilterProps> = ({
+  setGlobalFilter,
+  preGlobalFilteredRows,
+  globalFilter,
+}) => {
+  const count = preGlobalFilteredRows.length;
+
+  return (
+    <span>
+      <TextField
+        value={globalFilter || ''}
+        placeholder={`${count} records...`}
+        id="standard-basic"
```

Comment:
What is this id for? There may be more than one table per page, best not to give them IDs.